lib/debug/frame_info.rb

```diff
@@ -83,16 +82,23 @@ def frame_type
 
     def block_identifier
       return unless frame_type == :block
-      args = parameters_info
       _, level, block_loc = location.label.match(BLOCK_LABL_REGEXP).to_a
-      [level || "", block_loc, args]
+      [level || "", block_loc]
+    end
+
+    def block_args
+      return unless frame_type == :block
+      parameters_info
```

Comment:
so this and `method_args` actually just calls `parameters_info` 🤔 
I think the frame type check isn't very necessary here because they're not exposed to end users. how about we remove them and just call `parameters_info` from ThreadClient?